pytorch_lightning/trainer/trainer.py

```diff
@@ -2217,6 +2230,74 @@ def resume_from_checkpoint(self) -> Optional[Union[str, Path]]:
 
         return resume_from_checkpoint
 
+    @property
+    def ckpt_path(self) -> Optional[str]:
+        """Set to the path/URL of checkpoints loaded via :meth:`~pytorch_lightning.trainer.trainer.Trainer.fit`,
+        :meth:`~pytorch_lightning.trainer.trainer.Trainer.validate`,
+        :meth:`~pytorch_lightning.trainer.trainer.Trainer.test`, or
+        :meth:`~pytorch_lightning.trainer.trainer.Trainer.predict`. ``None`` otherwise."""
+        return self._ckpt_path
+
+    @property
+    def validated_ckpt_path(self) -> Optional[str]:
+        rank_zero_deprecation(
+            "The `Trainer.validated_ckpt_path` attribute was deprecated in v1.6 and will be removed in v1.8. The"
+            " path of checkpoints loaded via `Trainer.{fit,validate,test,predict}` should be accessed via"
```

Comment:
any reason you mention the plural of checkpoint everywhere here and in docstrings above? these attributes should reference exactly one or no checkpoint

Reply:
Thanks @awaelchli! The message here should have been more clear and I've updated it now accordingly.
I had the noun pluralized since it was referencing a set of objects (one potential checkpoint for each of the {fit,validate,test,predict} scenarios)... but while maybe grammatically correct, no one cares about that... we want a clear message! Thanks for the catch!

Reply:
awesome!